mlir/lib/Conversion/MIGraphXToTosa/MIGraphXToTosa.cpp

```diff
@@ -478,11 +498,47 @@ class ReduceMeanConverter final
   }
 };
 
+class QuantizeLinearConverter final
+    : public OpConversionPattern<migraphx::QuantizeLinearOp> {
+public:
+  using OpConversionPattern<migraphx::QuantizeLinearOp>::OpConversionPattern;
+
+  LogicalResult
+  matchAndRewrite(migraphx::QuantizeLinearOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const final {
+    auto input = op.getOperand(0);
```

Comment:
Nit: use actual names, not integer indices. Those are way too error-prone

Reply:
Yes, that's very reasonable.